docker-compose.yaml

```
@@ -0,0 +1,21 @@
+version: "3"
+services:
+  proxy:
+    image: nginx:1.17.4
+    volumes:
+      - ./nginx.conf:/etc/nginx/nginx.conf
+    ports:
+      - 8080:8080
```

You can listen on the 80 by default

```suggestion
      - 8080:80
```